cleanlab_studio/studio/inference.py

```diff
@@ -26,10 +30,15 @@
 
 
 class Model(abc.ABC):
-    """Base class for deployed model inference."""
+    """Represents a machine learning model instance in a Cleanlab Studio account.
+
+    Models should be instantiated using the [`Studio.get_model()`](../studio/#method-get_model) method. Then, using a Model object, you can [`predict()`](#method-predict) labels for new data.
+    """
 
     def __init__(self, api_key: str, model_id: str):
-        """Initializes model class w/ API key and model ID."""
+        """Initializes a model.
+
+        **Objects of this class are not meant to be constructed directly.** Instead, use [`Studio.get_model()`](../studio/#method-get_model)."""
```

Comment:
*add why you'd use get_model()

```suggestion
        **Objects of this class are not meant to be constructed directly.** Instead, use [`Studio.get_model()`](../studio/#method-get_model) to get a Model object."""
```